Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Orbital.cs

```diff
@@ -60,15 +60,15 @@ public Vector3 WorldOffset
 
         [SerializeField]
         [Tooltip("Lock the rotation to a specified number of steps around the tracked object.")]
-        private bool useAngleSteppingForWorldOffset = false;
+        private bool useAngleStepping = false;
```

Comment:
Please add the [`FormerlySerializedAs`](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html) tag to ensure we don't break previous serialization.